pino.js

```diff
@@ -100,6 +101,9 @@ function pino (...args) {
     edgeLimit
   } = opts
 
+  const major = Number(process.versions.node.split('.')[0])
+  if (major >= 14) warning.emit('PINODEP009')
```

Comment:
How is this limited to the usage of `pino.final`? This looks to me like it will warn on any usage of `pino()`.

Reply:
@jsumners  I was under the assumption that this was a one-time warning to be displayed when the logger is initialized rather than printing when`pino.final` actually gets called. I will change the code to move it to `final()`

Reply:
Thank you. If someone isn't directly using `pino.final` then this warning will be nothing more than an annoyance to them.

Reply:
good spot @jsumners